tools/brl_checks.c

```diff
@@ -190,7 +190,7 @@ check_base(const char *tableList, const char *input, const char *expected,
 			// Hm, something is not quite right. Try again with a larger outbuf
 			free(outbuf);
 			outlen = inlen * outlen_multiplier * (k + 1);
-			outbuf = malloc(sizeof(widechar) * outlen);
+			outbuf = malloc(sizeof(widechar) * (outlen + 1));
```

Comment:
Can you explain (in a comment) why the `+1`

Reply:
See above

Reply:
See above.